_posts/2021-01-05-Jamulus-Sound-Devices.md

```diff
@@ -265,7 +265,13 @@ _More testing required._
 
 **[Roland TD-27](https://www.roland.com/uk/products/td-27/)** USB drum sound module 
 
-**Windows**: ❌ Works (with provided ASIO driver, tested Windows 10), but **not** reccomended for Jamulus as you can't mute your direct sound.
+**Windows**: ✅ Works. Because the local master signal is always routed to the HeadphoneJack it's neccessary to change the signal-routing to hear your jamulus signal on the headphones and not the local signal of the drum modul. (Explanation: Search for "The Golden Rule" in the jamulus documentation):
```

Comment:
Even though we don't do in depth reviews,

```suggestion
**Windows**: ✅ Works. The local master signal is always routed to the Headphones. Using JACK is neccessary to change the signal-routing so that you hear your signal on the headphones - not the local signal of the drum module:
```